etc/osg/image-config.d/ce-common-startup

```diff
@@ -0,0 +1,6 @@
+# Return list of local users mapped from grid credentials
+function get_mapped_users () {
+    cat /etc/grid-security/grid-mapfile /etc/grid-security/voms-mapfile | \
+        awk '/^"[^"]+" +[a-zA-Z0-9\-\._]+$/ {print $NF}' | \
```

Comment:
It's OK how you have it, but excitingly, `|` (as with `&&` and `||`) will continue to the next line without needing a trailing `\`

```suggestion
    cat /etc/grid-security/grid-mapfile /etc/grid-security/voms-mapfile |
        awk '/^"[^"]+" +[a-zA-Z0-9\-\._]+$/ {print $NF}' |
```

Reply:
whatttt. life changing